synapse/tests/test_cmds_cortex.py

```diff
@@ -39,6 +39,20 @@ def getCoreCmdr(self, core):
         outp = s_output.OutPutStr()
         return s_cmdr.getItemCmdr(core, outp=outp)
 
+    def test_cmds_help(self):
```

Comment:
Added tests that fail before the bugfix is applied.  The bugfix makes them pass.